bin/phpunit.sh

```diff
@@ -4,4 +4,21 @@ if [[ $RUN_PHPCS == 1 || $SHOULD_DEPLOY == 1 ]]; then
 	exit
 fi
 
-if [ -f "phpunit.phar" ]; then php phpunit.phar -c phpunit.xml.dist $@; else ./vendor/bin/phpunit $@; fi;
+CURRENT_PHP_MAJOR_VERSION=$(php -r 'echo PHP_MAJOR_VERSION;')
+CURRENT_PHP_MINOR_VERSION=$(php -r 'echo PHP_MINOR_VERSION;')
+
+# The PHPUnit version inside composer.json is not compatible with PHP 7.2 and above
```

Comment:
You updated value `config.platform.php` in composer.json from `7.1` to `7.3` so this comment is no longer correct?

Reply:
I needed to bump the version because of the latest PHPUnit because the latest one doesn't support version below 7.3. If I didn't update this, you would receive an error that you couldn't install PHPUnit because of PHP version. But again, there is a different approach that I can use. I will push new commit to solve this.

Reply:
Thanks for the explanation 👍 !